activation/handler.go

```diff
@@ -176,28 +170,22 @@ func (h *Handler) SyntacticallyValidate(ctx context.Context, atx *types.Activati
 		if *atx.CommitmentATX == types.EmptyATXID {
 			return errors.New("empty commitment atx")
 		}
-		if atx.Sequence != 0 {
-			return errors.New("no prev atx declared, but sequence number not zero")
-		}
```

Comment:
V1 ATXs contain a sequence number and that number still needs to be verified to be monotonically increasing. We do not want to change validation behavior at all if we can avoid it.